packages/aws-pdk/src/aws_arch/src/index.ts

```diff
@@ -0,0 +1,5 @@
+/*! Copyright [Amazon.com](http://amazon.com/), Inc. or its affiliates. All Rights Reserved.
```

Comment:
I think you want to gitignore `packages/aws-pdk/src` :)

Reply:
Good catch - updated!